core/src/cljfmt/format/fn.clj

```diff
@@ -32,8 +32,7 @@
   (and (= :list (z/tag zloc))
        (let [form-sym (zl/form-symbol (z/down zloc))]
          (and (symbol? form-sym)
-              (or (= "fn" (name form-sym))
-                  (= "defn" (name form-sym))
+              (or (contains? #{"fn" "defn" "defn-"} (name form-sym))
```

Comment:
I think it's worth factoring this check out into a predicate like `fn-sym?` since it's used in three places in this ns.